app/charts/shared/interaction/tooltip.tsx

```diff
@@ -9,17 +10,35 @@ import {
 } from "@/charts/shared/interaction/tooltip-box";
 import {
   TooltipMultiple,
+  TooltipMultipleInverted,
   TooltipSingle,
+  TooltipSingleInverted,
 } from "@/charts/shared/interaction/tooltip-content";
 import { LegendSymbol } from "@/charts/shared/legend-color";
 import { useInteraction } from "@/charts/shared/use-interaction";
 import { Observation } from "@/domain/data";
 
-export const Tooltip = ({ type = "single" }: { type: TooltipType }) => {
+export const Tooltip = ({
+  type = "single",
+  inverted = false,
+}: {
+  type: TooltipType;
+  inverted?: boolean;
+}) => {
```

Comment:
As above, wondering if a change in property names would allow us to use the existing tooltip without `inverted` property?